samples/drivers/adc/adc_sequence/boards/stm32f3_disco.overlay

```diff
@@ -0,0 +1,141 @@
+/ {
```

Comment:
This file needs a license/copyright header

Reply:
Added